src/components/Dashboard/StudentData/index.js

```diff
@@ -280,6 +282,8 @@ function StudentData() {
 
   if (
     user.data.user.rolesList.indexOf("admin") > -1 ||
+    user.data.user.rolesList.indexOf("partner_view") > -1 ||
+    user.data.user.rolesList.indexOf("partner_edit") > -1 ||
```

Comment:
After importing the function:
```suggestion
    hasOneFrom(user.data.user.rolesList, ["admin", "partner_view", "partner_edit"]);
```